src/test/java/api/support/builders/ReorderQueueBuilder.java

```diff
@@ -0,0 +1,24 @@
+package api.support.builders;
+
+import org.folio.circulation.domain.reorder.ReorderRequest;
+import org.folio.circulation.domain.reorder.ReorderQueueRequest;
+
+import io.vertx.core.json.JsonObject;
+
+public class ReorderQueueBuilder implements Builder {
+  private final ReorderQueueRequest reorderQueueRequest = new ReorderQueueRequest();
+
+  @Override
+  public JsonObject create() {
+    return JsonObject.mapFrom(reorderQueueRequest);
```

Comment:
This appears to be the same mapping as used in the production code. What happens when this mapping is incorrect, how would we know?

Reply:
Jakson engine will throw an exception, something similar that we have when sending an undefined property to a storage module.